modules/machine-delete.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * machine_management/delete-machine.adoc
+
+[id="machine-delete_{context}"]
+= Deleting a specific machine
+
+You can delete specifics machine.
+
+.Prerequisites
+
+* Install an {product-title} cluster and the `oc` command line.
+* Log into `oc` as a user with `cluster-admin` permission.
+
+.Procedure
+
+. View the Machines that are in the cluster:
++
+----
+$ oc get machine -n openshift-machine-api
+----
++
+The Machines are listed in the form of `<clusterid>-worker-<cloud-region>`.
+
+. Delete the machine:
++
+----
+$ oc delete <machine> -n openshift-machine-api
+----
+
++
+[IMPORTANT]
+====
+By default, the machine controller tries to drain the node that is backed by the machine until it succeeds. In some situations, such as with a misconfigured pod disruption budget, the drain operation might not be able to succeed preventing the machine from being deleted. You can skip draining the node by setting "machine.openshift.io/exclude-node-draining" in a specific machine.
```

Comment:
> You can skip draining the node by setting "machine.openshift.io/exclude-node-draining" in a specific machine.

`s/setting/annotating/`, maybe use the word **annotating** instead of **setting**.